sps30-uart/sps30.c

```diff
@@ -148,6 +153,25 @@ int16_t sps30_read_measurement(struct sps30_measurement *measurement) {
     return 0;
 }
 
+int16_t sps30_sleep(void) {
+    struct sensirion_shdlc_rx_header header;
+
+    return sensirion_shdlc_xcv(SPS30_ADDR, SPS30_CMD_SLEEP, 0, NULL,
+                               0, &header, NULL);
+}
+
+int16_t sps30_wake_up(void) {
+    struct sensirion_shdlc_rx_header header;
+    int16_t ret;
+    const uint8_t data = 0xFF;
+    ret = sensirion_uart_tx(1, &data);
```

Comment:
would be style-consistent to have a blank line between declarations and code

Reply:
Fixed.